clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -60,15 +60,18 @@ object ClioClient extends LazyLogging {
 
     client
       .instanceMain(args)
-      .fold({
-        case UsageOrHelpAsked(message) => {
-          println(message)
-          Future.successful(())
-        }
-        case ParsingError(error) => {
-          Future.failed(new RuntimeException(error))
-        }
-      }, identity)
+      .fold(
+        {
+          case UsageOrHelpAsked(message) => {
+            println(message)
+            Future.successful(())
+          }
+          case ParsingOrAuthError(error) => {
+            Future.failed(new RuntimeException(error))
```

Comment:
I don't think a command line option error should result in an exception being thrown (which I think is what this will do?). We should print the error and exit with a non-zero status but not print an exception and a stack dump.